api/provider/aws/task_delete.go

```diff
@@ -7,6 +7,8 @@ import (
 	"github.com/aws/aws-sdk-go/service/ecs"
 )
 
+// Delete stops an ECS Task using the specified Task ID. The user's active cluster (Environment) is used as a filter
```

Comment:
There is no such thing as an "active cluster (Environment)". Layer0 has 0-to-many environments.

Reply:
Is it more accurate to say then that it retrieves a list of Task ARNs from all environments in the user's l0 instance?

Reply:
Yep! perfect